### PR TITLE
Add brand colors and xlarge padding token and increase logo clearspace

### LINKED ISSUE #777: Button padding
Buttons generally have quite large paddings on the left and right (see screenshot). Could be a bit more narrow. Is it intended like that or subject to change?
Screenshot 2020-05-28 at 14 22 52
Screenshot 2020-05-28 at 14 47 59

### LINKED ISSUE #784: Improve buttons design
To align with the overall oC design, here are some proposed values:

- Primary-button
```
background-color: #4E85C8;
border-radius: 3px;
font-weight: 600;
box-shadow: 0 4px 6px rgba(50,50,93,.11), 0 1px 3px rgba(0,0,0,.08);
```
- Default-button
```
border-color: #4E85C8;
border-width: 1px;
border-radius: 3px;
color: #4E85C8;
font-weight: 600;
```
- Raw-button / link
```
color: #4E85C8;
:hover
text-decoration: underline;
```

### LINKED ISSUE #785: Use oC CI font
oC has defined Source Sans Pro as the font we are using. We need to bring it into the default theme.

### LINKED ISSUE #796: Primary button is one pixel smaller in height
In the new button design, when having a primary button and a button without drop shadow side by side, it feels like they have an offset / are not vertical aligned. Looking at the pixels, you can see that the primary button is one pixel smaller in height.

Screenshot 2020-06-16 at 11 10 26

Screenshot 2020-06-16 at 11 32 55

### PR DESCRIPTION
**Pull request by LukasHirt**
Bring oC CI into the default theme, improve buttons design and improve the size of the logo in the sidebar. This PR contains quite a few changes but they're quite small in size so I merged them into one PR.

## Related issues:
- refs #786
- refs #546
- closes #785
- closes #784 
- closes #777
- Closes #796

### CODE REVIEW

File: src/styles/theme/oc-button.scss [hunk 1]
```diff
@@ -39,17 +69,83 @@
...
+
+    .oc-icon > svg {
+      fill: $action-primary;
+    }
+  }
+
+  &.uk-button-primary {
+    border: none;
+    background-color: $action-primary;
+    box-shadow: 0 4px 6px rgba($shadow-action-primary, .11), 0 1px 3px rgba($shadow-global, .08);
```
Comment (micbar) [inline src/styles/theme/oc-button.scss:108]: Codacy Issue found: [Expected a leading zero (number-leading-zero)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5696666)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 7
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5696666&bid=18647091)

File: src/styles/theme/oc-button.scss [hunk 2]
```diff
@@ -39,17 +69,83 @@
...
+      background-color: $action-primary-hover;
+    }
+  }
 }
 
 .oc-button.uk-button-raw {
   @extend .oc-button-reset;
 }
+
+.uk-button-group .oc-button {
```
Comment (micbar) [inline src/styles/theme/oc-button.scss:122]: Codacy Issue found: [Expected selector ".uk-button-group .oc-button" to come before selector ".oc-button:not(.uk-button-small):not(.uk-button-large):not(.uk-button-raw)" (no-descending-specificity)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5696666)

File: src/styles/theme/oc-button.scss [hunk 3]
```diff
@@ -39,17 +69,83 @@
...
+    &:focus {
+      border-color: $action-primary-hover;
+      color: $action-primary-hover;
+
+      .oc-icon > svg {
+        fill: $action-primary-hover;
+      }
+    }
+
+    .oc-icon > svg {
```
Comment (micbar) [inline src/styles/theme/oc-button.scss:100]: Codacy Issue found: [Expected selector ".oc-button.uk-button-default .oc-icon > svg" to come before selector ".oc-button.uk-button-default:hover .oc-icon > svg" (no-descending-specificity)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5696666)

File: src/styles/theme/oc-button.scss [hunk 4]
```diff
@@ -39,17 +69,83 @@
...
+  }
+
+  &.uk-button-default {
+    border-left: none;
+    border-right: none;
+
+    &:first-of-type {
+      border-left: 1px solid $action-primary;
+    }
+
```
Comment (micbar) [inline src/styles/theme/oc-button.scss:142]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5696666)

File: src/styles/theme/oc-button.scss [hunk 5]
```diff
@@ -39,17 +69,83 @@
...
 .oc-button.uk-button-raw {
   @extend .oc-button-reset;
 }
+
+.uk-button-group .oc-button {
+  border-radius: 0;
+  // Prevent negative margin which is coming from UIkit
+  margin-left: 0 !important;
+
+  &:first-of-type {
```
Comment (micbar) [inline src/styles/theme/oc-button.scss:127]: Codacy Issue found: [Expected selector ".uk-button-group .oc-button:first-of-type" to come before selector ".oc-button:not(.uk-button-small):not(.uk-button-large):not(.uk-button-raw)" (no-descending-specificity)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5696666)

File: src/styles/theme/oc-button.scss [hunk 6]
```diff
@@ -4,13 +4,39 @@
...
+  flex-flow: row wrap;
+  border: none;
+  border-radius: 3px;
+  padding: 0.5rem 0.75rem;
+  font-size: 1.125rem;
+  font-weight: 600;
+  line-height: 1.4;
+  justify-content: center;
+  align-items: center;
+  transition: background-color $transition-duration-short ease-in-out,
```
Comment (micbar) [inline src/styles/theme/oc-button.scss:17]: Codacy Issue found: [Expected newline after ":" with a multi-line declaration (declaration-colon-newline-after)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5696666)

File: src/styles/theme/oc-button.scss [hunk 7]
```diff
@@ -39,17 +69,83 @@
...
+.uk-button-group .oc-button {
+  border-radius: 0;
+  // Prevent negative margin which is coming from UIkit
+  margin-left: 0 !important;
+
+  &:first-of-type {
+    border-radius: 3px 0 0 3px;
+  }
+
+  &:last-of-type {
```
Comment (micbar) [inline src/styles/theme/oc-button.scss:131]: Codacy Issue found: [Expected selector ".uk-button-group .oc-button:last-of-type" to come before selector ".oc-button:not(.uk-button-small):not(.uk-button-large):not(.uk-button-raw)" (no-descending-specificity)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5696666)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5696666&bid=18647091)

File: src/styles/theme/oc-button.scss [hunk 8]
```diff
@@ -39,15 +64,85 @@
...
+    &:focus {
+      background-color: $action-primary-hover;
+    }
+  }
+
+  .uk-button-group & {
+    border-radius: 0;
+    // Prevent negative margin which is coming from UIkit
+    margin-left: 0 !important;
+
```
Comment (micbar) [inline src/styles/theme/oc-button.scss:116]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5696666)

**Comment by micbar** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5696666&bid=18647091)

File: src/styles/theme/oc-button.scss [hunk 9]
```diff
@@ -39,15 +64,85 @@
...
+    &:first-of-type {
+      border-radius: 3px 0 0 3px;
+    }
+
```
Comment (micbar) [inline src/styles/theme/oc-button.scss:120]: Codacy Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/owncloud/owncloud-design-system/pullRequest?prid=5696666)

**Comment by kulmann** [comment]
Turns out that one of  my two concerns is actually a small bug: https://github.com/owncloud/owncloud-design-system/issues/796
Please decide if you want to fix that in a separate PR

This is my other concern: https://github.com/owncloud/owncloud-design-system/issues/795
Should be evaluated through user feedback.

**Comment by LukasHirt** [comment]
Different buttons size fixed by using the transparent border in primary, secondary and danger button (falling back to default in UIkit).

Closes #796

**Pull request merged**